src/utils/version_helper.py

```diff
@@ -18,3 +20,22 @@ def is_valid_sonar_version(version: str) -> bool:
             return False
 
         return True
+
+    @staticmethod
+    def is_valid_plus_signed_version(version: str) -> bool:
+
+        # This is an explicit requirement for project SLVSCODE
+        # see https://sonarsource.atlassian.net/browse/BUILD-4915 for more details
+
+        if "+" not in version:
+            return False
+
+        return VersionHelper.is_valid_sonar_version(VersionHelper._sanitize_version(version))
+
+    @staticmethod
+    def is_valid_version(version: str) -> bool:
+        return VersionHelper.is_valid_sonar_version(version) or VersionHelper.is_valid_plus_signed_version(version)
+
+    @staticmethod
+    def _sanitize_version(version: str) -> str:
+        return version.replace("+", ".")
```

Comment:
Could this be more specific and only apply to the last separator character? 
`<version> ::= <major> "." <minor> "." <patch> ".|+" <build>`
I'm adding a suggestion to test this.

Reply:
That could be done for sure, I will look into that thank you for the remark :+1:

Reply:
Implemented with ac64010ff0fe404a0c25fad4f8537034664b9c99